modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -138,6 +139,7 @@ public TypeScriptAngularClientCodegen() {
         this.cliOptions.add(new CliOption(FILE_NAMING, "Naming convention for the output files: 'camelCase', 'kebab-case'.").defaultValue(this.fileNaming));
         this.cliOptions.add(new CliOption(STRING_ENUMS, STRING_ENUMS_DESC).defaultValue(String.valueOf(this.stringEnums)));
         this.cliOptions.add(new CliOption(QUERY_PARAM_OBJECT_FORMAT, "The format for query param objects: 'dot', 'json', 'key'.").defaultValue(this.queryParamObjectFormat.name()));
+        this.cliOptions.add(CliOption.newBoolean(USE_SQUARE_BRACKETS_IN_ARRAY_NAMES, "Setting this property to true will add brackets to array attribute names.", false));
```

Comment:
```suggestion
        this.cliOptions.add(CliOption.newBoolean(USE_SQUARE_BRACKETS_IN_ARRAY_NAMES, "Setting this property to true will add brackets to array attribute names, e.g. my_values[].", false));
```

Reply:
> 

Hello @macjohnny,

I updated the description as you suggested and the doc has been updated.
Please have a look.

Than you in advance :)

Reply:
Hello @macjohnny,

Thank you for the review, I just pushed an update for TypeScriptAngularClientOptionsTest file
to fix a broken test case. The build is now passing, please can you review the PR again ?

Thank you in advance.

Reply:
Thanks, if the CI now passes, we can merge